sources/README.md

```diff
@@ -862,6 +862,18 @@ _A gazetteer of non-overlapping, authoritative polygons around a curated list of
 * remarks _https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/quattroshapes_remarks.md_
 * license _https://github.com/foursquare/quattroshapes/blob/master/LICENSE.md_
 
+## Quattroshapes Point Gazetteer
+
+_The Quattroshapes point gazetteer. A gazetteer of non-overlapping, authoritative polygons around a curated list of places._
```

Comment:
`The Quattroshapes point gazetteer. A  big list of point locations that supplements the polygons gazetteer.`